### PR TITLE
negative mass bugfix

### LINKED ISSUE #631: mass parameter has no lower-limit
The mass parameter (generally constrained, but can be flipped to be an input parameter) does not have a defined lower-limit, which allows negative values to be set, causing other constraints to resolve to nans.

See [#630](https://github.com/phoebe-project/phoebe2/discussions/630) for an example.

### PR DESCRIPTION
**Pull request by MichaelAbdul-Masih**
fixed a bug where the mass could be set to negative values.

Fixes #631

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def _env_variable_bool(key, default):
     long_description = "\n".join(long_description_s[long_description_s.index("INTRODUCTION"):])
 
 setup (name = 'phoebe',
-       version = '2.4.4',
+       version = '2.4.5',
        description = 'PHOEBE devel version',
```
Comment (kecnry) [inline setup.py:409]: while we're at it, can you update this?

```suggestion
       description = 'PHOEBE 2.4',
```

File: phoebe/frontend/default_bundles/default_binary.bundle [hunk 2]
```diff
@@ -426,8 +426,8 @@ null
 "description": "Source for bolometric limb darkening coefficients (used only for irradiation; 'auto' to interpolate from the applicable table according to the 'atm' parameter, or the name of a specific atmosphere table)",
 "choices": [
 "auto",
-"phoenix",
-"ck2004"
+"ck2004",
+"phoenix"
```
Comment (aprsa) [inline phoebe/frontend/default_bundles/default_binary.bundle:430]: Should these edits be a part of this particular bugfix?

File: phoebe/frontend/default_bundles/default_binary.bundle [hunk 3]
```diff
@@ -463,7 +463,7 @@ null
 "value": 0.9988131358058301,
 "default_unit": "solMass",
 "limits": [
-null,
+0.0,
```
Comment (aprsa) [inline phoebe/frontend/default_bundles/default_binary.bundle:466]: Should this also be set to 1e-12, to be consistent with Kyle's proposed edit in the constraints?

File: phoebe/frontend/default_bundles/default_contact_binary.bundle [hunk 4]
```diff
@@ -463,7 +463,7 @@ null
 "value": 1.0089067994531355,
 "default_unit": "solMass",
 "limits": [
-null,
+0.0,
```
Comment (aprsa) [inline phoebe/frontend/default_bundles/default_contact_binary.bundle:466]: Same as above: 1e-12?

File: phoebe/frontend/default_bundles/default_contact_binary.bundle [hunk 5]
```diff
@@ -811,7 +811,7 @@ null
...
```
Comment (aprsa) [inline phoebe/frontend/default_bundles/default_contact_binary.bundle:814]: Same as above: 1e-12?

File: phoebe/frontend/default_bundles/default_star.bundle [hunk 6]
```diff
@@ -463,7 +463,7 @@ null
 "value": 1.0,
 "default_unit": "solMass",
 "limits": [
-null,
+0.0,
```
Comment (aprsa) [inline phoebe/frontend/default_bundles/default_star.bundle:466]: Same as above: 1e-12?

**Review by aprsa — CHANGES_REQUESTED** [review]

File: phoebe/frontend/default_bundles/default_binary.bundle (hunk 2 above)
Comment (kecnry) [reply phoebe/frontend/default_bundles/default_binary.bundle:430]: @MichaelAbdul-Masih - these can safely be excluded from the diff if you want, but also don't do any harm (I think the order when exporting to json is somewhat random and so this often shows up when regenerating the default cached bundles).

**Review by kecnry — APPROVED** [review]
Looks good to me, thanks!  I'll merge/release once CI passes (it didn't run automatically, but it turns out the triggers weren't setup for a push to a PR, so I updated that and rebased this branch to force it to rerun)

**Pull request merged**